packml_stacklight/src/packml_stacklight.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Software License Agreement (Apache License)
+ *
+ * Copyright (c) 2019 Joshua Hatzenbuehler
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "packml_stacklight/packml_stacklight.h"
+#include "packml_stacklight/utils.h"
+
+namespace packml_stacklight
+{
+PackmlStacklight::PackmlStacklight(ros::NodeHandle nh, ros::NodeHandle pn) : nh_(nh), pn_(pn)
+{
+  ros::NodeHandle packml_node("~/packml");
+
+  status_sub_ = packml_node.subscribe<packml_msgs::Status>("status", 1, &PackmlStacklight::callBackStatus, this,
```

Comment:
Any subscribers should use the passed in `nh`

Reply:
`ros::TransportHints().reliable().tcpNoDelay(true)` should only be used on high frequency topics.